packages/approval/doc/classes/actionapi.md

```diff
@@ -38,7 +38,7 @@ ActionApi - object-oriented interface
 
 *Inherited from [BaseAPI](baseapi.md).[constructor](baseapi.md#constructor)*
 
-*Defined in [packages/approval/base.ts:49](https://github.com/RedHatInsights/javascript-clients/blob/master/packages/approval/base.ts#L49)*
+*Defined in [packages/approval/base.ts:49](https://github.com/katerinapat/javascript-clients/blob/master/packages/approval/base.ts#L49)*
```

Comment:
This will break the docs. Can you switch remote to upstream when generating the clients?

Reply:
resolved it